src/bin/rad-ci.rs

```diff
@@ -0,0 +1,161 @@
+use std::convert::TryInto;
+use std::io::{Read, Write};
+use std::net::TcpStream;
+
+use rad_ci::{Options, HELP};
+use rad_terminal::components as term;
+
+use anyhow::Context;
+use ssh2::Session;
+
+fn main() {
+    rad_terminal::args::run_command::<Options, _>(HELP, "CI", run);
+}
+
+fn run(options: Options) -> anyhow::Result<()> {
+    let seed = options.seed.context("Seed is invalid")?;
+    // SSH into server
+    let tcp = TcpStream::connect(format!("{}:22", &seed))?;
+    let mut sess = Session::new()?;
+    sess.set_tcp_stream(tcp);
+    sess.handshake()?;
+
+    // Authenticate
+    let spinner = term::spinner(&format!("SSH into remote {}", &seed));
```

Comment:
Spinner should probably start before the TCP stream is connected.

Reply:
yup https://github.com/radicle-dev/radicle-client-tools/pull/35/commits/21aa4f338dfbf8b7570287a4f7701d4462f48278#diff-2f5300f7dedf306f4b73a94a6c5c3674f3ece69ed4bda39a3b27b8448149d978R122